sql_2g/report_queries/erm/count/erm_agreement_count.sql

```diff
@@ -8,16 +8,67 @@ TABLES
 folio_agreements.refdata_value 
 
 */
+
+WITH parameters AS (
+    SELECT
+    
+		-- filters on agreement level
+		''::VARCHAR AS agreement_status, -- Enter your subscription agreement status eg. 'Active', 'Closed' etc.
+		-- subscription agreement time period will be added when available (IRIS)
+		--NULL::DATE AS a_start_date,
+        --NULL::DATE AS a_end_date,
+        --'2021-01-01' :: DATE AS sa_start_date, -- start date day is included in interval
+        --'2022-01-01' :: DATE AS sa_end_date, -- end date day is NOT included in interval -> enter next day		
+		-- filters on erm_resource level
+		''::VARCHAR AS resource_type, -- Enter your erm resource type eg. 'monograph', 'serial' etc.
+		-- filters on entitelment level
+        NULL::DATE AS ent_start_date,
+        NULL::DATE AS ent_end_date,
+        --'2021-01-01' :: DATE AS ent_start_date, -- start date day is included in interval
+        --'2022-01-01' :: DATE AS ent_end_date, -- end date day is NOT included in interval -> enter next day	
+		-- filters on package content item level
+        NULL::DATE AS pci_start_date,
+        NULL::DATE AS pci_end_date,
```

Comment:
extra comma at the end of line should be removed

Reply:
Thanks, have it removed